### DISCUSSION TITLE
WoD20 Willpower

### DISCUSSION THREAD

**Discussion by mariuskane** [JohanFalt/Foundry_WoD20#865] [Q&A] [answered]
Question as per the V/W20 rule

• Automatic Successes: Spending a Willpower point
on an action gives the player one extra success on any
roll. Only one point can be spent this way each turn, but
the success is guaranteed. Spending Willpower in this
way completely negates the effects of a botch. Some rolls
do not allow a character to spend Willpower, including
damage rolls or any roll to activate Gifts.

Is there a way to do this in the module as I can't find a tickbox to be able to force this guaranteed success.

**Comment by JohanFalt** [comment, marked as answer]
I'll make it an issue
